src/marvin/cli/server.py

```diff
@@ -14,6 +14,12 @@ def start(
         "-p",
         help="Port to run the server on",
     ),
+    host: str = typer.Option(
```

Comment:
let's add a default here from the marvin settings
```suggestion
    host: str = typer.Option(
        marvin.settings.api_base_url,
```